reach4help/reach4help
# Development

## Description
Adds Latest code from developement branch into the master branch.
New Features:
1. An alternate List view of all requests in the find request page for volunteers.
2. Newer Sign Up Screens
3. Changes in Translation
## Motivation

## Testing Guidelines
1. Make sure that the pages are all translated properly for languages other than english and nothing similar to `module.personal_data` shows up on any screen for languages other than english (we support spanish, portughese, french) (emphasis on the sign up pages, personal data pages, new request page and the find request page)
2. Ensure sign up is flawless for Google and Facebook
3. Ensure Language change is possible from the landing page and persists across session and isn't based on user's default browser language
4. The process of making an offer for a request works perfectly fine for both the list view and the map view
5. The process of rejecting a request works perfectly fine for both the list view and the map view
6. The process of making a request works perfectly fine
7. The redirections and refresh states are all correct for points 4, 5 and 6 above
## Release Checklist


## Security Checklist

This PR creates, modifies, or deletes:




## Additional Notes
